README.md

```diff
@@ -185,6 +185,41 @@ eval $(ssh-agent)
 ssh-add
 ```
 
+#### The Things Network Requirements
+
+To be able to run the automatic tests in spec [11-lorawan](11-lorawan/test_spec11.py) a valid TTN
+account is required, and an application needs to be created with a device configure
+to do `OTAA` and another configured to do `ABP`.
+
+- [Create a TTN account]
+- [Add an application]. Take note of:
+  - Application ID as `TTN_APP_ID`
+- [Register your device], one for `ABP` and a second for `OTAA`. Take note
+  of:
+  - ABP device ID as `TTN_DEV_ID_ABP`
+  - OTAA device ID as `TTN_DEV_ID`
+- Personalize a first device for OTAA (this is the default configuration).
+  Take note of:
+  - Device EUI as `DEVEUI`
+  - Application EUI as `APPEUI`
+  - App Key as `APPKEY`
+- [Personalize device for ABP] and disable `Frame counter checks` to avoid
```

Comment:
I don't have a view of the page (since I don't have clicked myself a TTN account yet), but I believe as a quote from a web page

```suggestion
- [Personalize device for ABP] and disable **Frame counter checks** to avoid
```

is more appropriate.